### PR TITLE
Fall 2020/corgi

### PR DESCRIPTION
**Pull request by aprasad888**

### CODE REVIEW

File: cohorts/3/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <img src = "https://camo.githubusercontent.com/504e0397da3d336b7292a8eae06a83caf90d490b/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f63612f37382f39392f63613738393933613436386138333531393762326533343064633231346364312e6a7067">
+        <h3>
+            License
+        </h3>
+        <p>
+            This repository is licensed under <a href = "https://github.com/GTBitsOfGood/bootcamp/blob/fall-2020/corgi/LICENSE.txt">GNU GPLv3 License</a>.
+        </p>
+    </main>
+</body>
+<footer>
```
Comment (whymarlene) [inline cohorts/3/index.html:36]: If you don't have a footer, then there is no need to have the tag.

File: README.md [hunk 2]
```diff
@@ -5,4 +5,4 @@ The Bits of Good Bootcamp is a MERN stack bootcamp that gives Georgia Tech stude
```
Comment (whymarlene) [inline README.md]: I believe we were told to remove the README in our cohort's repo.

File: cohorts/3/index.html (hunk 1 above)
Comment (whymarlene) [inline cohorts/3/index.html]: If you don't have a footer, then there is no need to have the tag.

Comment (whymarlene) [inline cohorts/3/index.html]: Footer (and header) tags go inside the body.

**Review by whymarlene — COMMENTED** [review]
For the most part, it looks good! Make sure to remove the README file that is outside your cohort's folder.

**Review by Arthelon — APPROVED** [review]

**Pull request merged**